src/types/Movie.ts

```diff
@@ -1,5 +1,7 @@
 export interface Movie {
   title: string;
-
-  // add all the other field
+  description: string;
```

Comment:
Consider using more descriptive names for fields. `imgUrl` and `imdbUrl` could be `imageURL` and `imdbURL` respectively. 🤔